alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -154,7 +153,7 @@ pub fn new(config: &Options, window_size: WindowSize) -> Option<Pty> {
 
         // This call was expected to return false.
         if failure {
-            panic_shell_spawn();
+            return Err(std::io::Error::last_os_error());
```

Comment:
Should import at least `std::io` for all of these to keep the paths a little under control.

Reply:
Makes sense, thank you!